examples/fizz-buzz.ktn

```diff
@@ -0,0 +1,26 @@
+define buzzes(Int32 -> +IO +Fail):
+    -> n;
+    n 5 (%) 0 (=)
+    n 3 (%) 0 (=)
+
+    match
+    case true: 
+        match 
+        case true:
+            "FizzBuzz" say
+        case false:
+            "Fizz" say
+    case false:
+        match
+        case true:
+            "Buzz" say
+        case false:
+            n say
```

Comment:
Since all branches end with `say`, you can factor it out to after the `match`/`if` if you change `n say` to `n show` so every branch returns a string. But moreover, you can remove the `+IO` constraint by moving the `say` out of `buzzes` and into `loop`, changing the return type of `buzzes` to `List<Char>`.